src/lib/registration/clients/inst_scc.rb

```diff
@@ -294,7 +308,11 @@ def registration_ui
     def workflow_aliases
       {
         # skip this when going back
-        "check"                  => ->() { registration_check },
+        "check"                  => lambda do
```

Comment:
Looking at this, and at the added `log.group`s in `pkg_manager` and `register_addons`... it seems obvious that we should tell `Sequencer.Run` to make a group for each *alias* it runs, right?

Reply:
Um, that would be nice, but unfortunately there is no human readable description of each step which I'd like to have in the log. We would need to extend the logic and the workflow data...